src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -1,6 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0006</DiagnosticId>
+    <Target>M:System.Numerics.IBinaryInteger`1.DivRem(`0,`0,System.DivisionRounding)</Target>
+    <Left>net8.0/System.Runtime.dll</Left>
+    <Right>net9.0/System.Runtime.dll</Right>
+  </Suppression>
```

Comment:
This suppression shouldn't be required if you are creating the ref source correctly.

Reply:
I've removed the suppression, I was able to build with `build.cmd -configuration release /p:ApiCompatValidateAssemblies=false`